### PR TITLE
Implemented code and testing for task 8 PATCH and also refactored cod…

### PR DESCRIPTION
**Pull request by beckylakes**
…e and added tests for task 7 POST comment

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -293,3 +334,74 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+  });
+  test("should respond with a 400 status code if article_id is invalid", () => {
+    return request(app)
+      .patch("/api/articles/not-an-article")
+      .send({ inc_votes: 1 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
```
Comment (ghost) [inline __tests__/app.test.js:406]: What about if the key inc_votes is missing or not a number?

File: models/articles.model.js [hunk 2]
```diff
@@ -22,9 +22,24 @@ SELECT articles.article_id, articles.author, articles.title, articles.topic, art
     )
     .then((result) => {
       return result.rows;
-    }).catch((err) => {
-      console.log(err)
+    })
+    .catch((err) => {
+      console.log(err);
```
Comment (ghost) [inline models/articles.model.js:27]: Console logging an error is not ideal, this should be sending to the next error handlers.

**Review by ghost — COMMENTED** [review]
Almost there, a few more tests to handle those edge cases.
